pages/about.html

```diff
@@ -1,12 +1,73 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Sana</title>
+    <link href="../styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+    <header>  
+        <h1>A Little About ME!</h1>
+        <div class="top">
+            <a href="index.html"> Swiftie Website</a>
+            <a href="portfolio.html"> My Portfolio</a>
+        </div>
+    </header>
+
+<section>
+    <div class="intro">
+        
+        <img alt="A picture of me on a kayak laughing" src="../assets/my_photo.png"/>
+        
+        <div>
+            <h1> Hi I am Sana!</h1> 
```

Comment:
It depends on who you ask, but your document should have only one h1 element. Of course you can add multiple and the browser will still render the h1 tags, but it is more semantically correct to only use 1 since heading tags have a hierarchy

Since you already have an h1 element on line 12, you can use a h2 here.